include/ensmallen_bits/gradient_descent/gradient_descent.hpp

```diff
@@ -45,11 +45,10 @@ class GradientDescent
  public:
   /**
    * Construct the Gradient Descent optimizer with the given function and
-   * parameters.  The defaults here are not necessarily good for the given
+   * parameters. The defaults here are not necessarily good for the given
```

Comment:
Can we not make these changes everywhere?  It is okay to use one or two spaces (i.e., the world has not decided which of these is right and which of these isn't).  So let's leave it how it is.